calliope/backend/run.py

```diff
@@ -121,6 +129,115 @@ def run_plan(model_data, timings, backend, build_only, backend_rerun=False):
     return results, backend_model
 
 
+def run_spores(model_data, timings, interface, backend, build_only):
+    """
+    For use when mode is 'spores', to allow the model to be built, edited, and
+    iteratively run within Pyomo, modifying, at each iteration, the score of
+    each loc::tech in such a way to generate Spatially explicit Practically Optimal
+    RESults (SPORES).
+
+    """
+    log_time(logger, timings, 'run_start',
+             comment='Backend: starting model run in SPORES mode')
+
+    run_config = UpdateObserverDict(
+        initial_yaml_string=model_data.attrs['run_config'], name='run_config', observer=model_data
+    )
+
+    n_spores = 1  # TODO: turn into configurable in run_config
```

Comment:
There needs to be configurability, which is best within the 'run' configuration settings. E.g. under 'spores_options'.

Reply:
Sure, this was in my TODO list already